src/components/app/MobileWorkBoard.jsx

```
@@ -40,54 +43,73 @@ const MobileWorkBoard = ({
+      setShowLoginAlert(true)
+      return
+    }
+    try {
+      await toggleScrap(postId, type)
+      const nowScrapped = Boolean(useScrapStore.getState().scraps[postId])
+      if (nowScrapped) {
+        ToastService.success('스크랩되었습니다.')
+      } else {
+        ToastService.info('스크랩이 해제되었습니다.')
+      }
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(no-unused-vars)